webserver/views/datasets.py

```diff
@@ -183,7 +188,7 @@ def edit(dataset_id):
             "datasets/edit.html",
             mode="edit",
             dataset_id=str(dataset_id),
-            dataset_name=ds["name"],
```

Comment:
You didn't need to remove the comma. Useful tip I've seen before: http://dave.cheney.net/2014/10/04/that-trailing-comma. :)